freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/AvailabilityType.java

```diff
@@ -8,6 +8,7 @@
 
 public enum AvailabilityType {
 
+    TEMPORARY_FOUR_NODE_BASED(4),
```

Comment:
while this should work for now I think this is a bit fragile. What happens if we have 5 instance somehow? I think we should have some generic solution to handle the case which is an unsupported scenario. And if we start supporting 4 node based installation then it would be a new valid config here while the error handling would be the same

Reply:
just an idea as it would require some more code modification and mappings but something like `TEMPORARY(0)` could work